cmd/kyma/completion/completion.go

```diff
@@ -1,20 +1,30 @@
 package completion
 
 import (
+	"fmt"
+	"io"
 	"os"
 
+	"github.com/pkg/errors"
+
 	"github.com/spf13/cobra"
 )
 
-//NewCompletionCmd creates a new completion command
+//NewCmd creates a new completion command
 func NewCmd() *cobra.Command {
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
-		Short: "Generates bash completion scripts.",
+		Use:   "completion bash|zsh",
+		Short: "Generates bash or zsh completion scripts",
 		Long: `Use this command to display the shell completion code used for interactive command completion. 
-To configure your bash shell to load completions, add ` + "`. <(kyma completion)`" + ` to your bash profile.
-
- 
+		To configure your shell to load completions, add ` + "`. <(kyma completion bash)`" + ` to your bash profile or ` + "`. <(kyma completion zsh)`" + ` to your zsh profile.
```

Comment:
```suggestion
To configure your shell to load completions, add ` + "`. <(kyma completion bash)`" + ` to your bash profile or ` + "`. <(kyma completion zsh)`" + ` to your zsh profile.
```
Is the indentation intentional?